variables.tf

```diff
@@ -21,16 +21,34 @@ variable "enable_account_best_practices" {
 }
 
 variable "nat" {
-  description = "Select NO_NAT for no NAT, NAT_INSTANCES for NAT based on EC2 instances, or NAT_GATEWAY for NAT with AWS NAT Gateways."
+  description = "Select NO_NAT for no NAT, NAT_INSTANCES for NAT based on EC2 instances, NAT_GATEWAY for NAT with AWS NAT Gateways, or ALTER_NAT for hourly based EC2 instances."
   type        = string
   default     = "NO_NAT"
 
   validation {
-    condition     = contains(["NAT_INSTANCES", "NO_NAT", "NAT_GATEWAY_PER_SUBNET", "NAT_GATEWAY_SINGLE", "NAT_GATEWAY_PER_AZ"], var.nat)
-    error_message = "Only 'NO_NAT', 'NAT_INSTANCES', 'NAT_GATEWAY_PER_SUBNET', 'NAT_GATEWAY_SINGLE' and 'NAT_GATEWAY_PER_AZ' are allowed."
+    condition     = contains(["NAT_INSTANCES", "NO_NAT", "NAT_GATEWAY_PER_SUBNET", "NAT_GATEWAY_SINGLE", "NAT_GATEWAY_PER_AZ", "ALTER_NAT"], var.nat)
+    error_message = "Only 'NO_NAT', 'NAT_INSTANCES', 'NAT_GATEWAY_PER_SUBNET', 'NAT_GATEWAY_SINGLE', 'NAT_GATEWAY_PER_AZ', and 'ALTER_NAT' are allowed."
   }
 }
 
+variable "ingress_security_group_ids" {
```

Comment:
add prefix "alternat_" to variable name so that it becomes clear that this belongs to the alternat option.